BUILD.md

```diff
@@ -61,6 +61,9 @@ The complete list of build options can be found by running `./build.sh (or ./bui
 
 1. In the ONNX Runtime root folder, run `./build.sh --config RelWithDebInfo --build_server  --use_openmp --parallel`
 
+ONNX Runtime server requires you to have Go installed to build the server. 
```

Comment:
NIT: Please make it the `1.` and move above the build command. Also we may want to clarify that only build ONNX Runtime Server needs to install golang.